elastic/security/templates/component/logs-endpoint.events.library@package.json

```
@@ -4,9 +4,13 @@
     "template": {
       "settings": {
         "index": {
-          "lifecycle": {
-            "name": "logs"
-          },
+          {{% if lifecycle == "ilm" or (not lifecycle and build_flavor == "default") %}
```

double curly brackets
```suggestion
          {% if lifecycle == "ilm" or (not lifecycle and build_flavor == "default") %}
```

---

👍  Thanks Gareth...